src/ctapipe/resources/stats_calc_config.yaml

```diff
@@ -0,0 +1,37 @@
+StatisticsCalculatorTool:
+  allowed_tels: [1,2,3,4]
+  dl1a_column_name: "image"
+  output_column_name: "statistics"
+
+PixelStatisticsCalculator:
```

Comment:
I think you want to make this example a bit more complex and apply to the telescopes of a different kind (i.e. with different parameters of the PixelStatisticsCalculator)